pkg/server/handlers.go

```diff
@@ -30,6 +35,64 @@ func HostsHandler(w http.ResponseWriter, r *http.Request) {
 	RespondWithJson(w, hostList)
 }
 
+type InstallConfig struct {
+	ClusterName string `json:"clusterName"`
+	DNSDomain   string `json:"DNSDomain"`
+	PullSecret  string `json:"pullSecret"`
+}
+
+type DefineClusterCredentials struct {
+	ClusterName string `json:"clusterName"`
+	DNSDomain   string `json:"DNSDomain"`
+	Username    string `json:"username"`
+	Password    string `json:"password"`
+}
+
+func CreateInstallConfig(w http.ResponseWriter, r *http.Request) {
+	var clusterDefinition DefineClusterCredentials
+	_ = json.NewDecoder(r.Body).Decode(&clusterDefinition)
+
+	tokenRequestURL := fmt.Sprintf("https://developers.redhat.com/auth/realms/rhd/protocol/openid-connect/token")
```

Comment:
`fmt.Sprintf` seems unnecessary here --- is this a left-over?

Also, the URL could be a constant at the top of the file?